module/VuFind/config/module.config.php

```diff
@@ -800,9 +800,27 @@
 
 // Define non tab record actions
 $nonTabRecordActions = [
-    'AddComment', 'DeleteComment', 'AddTag', 'DeleteTag', 'Save', 'Email', 'SMS',
-    'Cite', 'Explain', 'Export', 'RDF', 'Hold', 'Home', 'StorageRetrievalRequest',
-    'AjaxTab', 'ILLRequest', 'PDF', 'Epub', 'LinkedText', 'Permalink', 'Rating',
+    'AddComment',
+    'DeleteComment',
+    'AddTag',
+    'DeleteTag',
+    'Save',
+    'Email',
+    'SMS',
+    'Cite',
+    'Explain',
+    'Export',
+    'RDF',
+    'Hold',
+    'Home',
+    'StorageRetrievalRequest',
+    'AjaxTab',
+    'ILLRequest',
+    'PDF',
+    'Epub',
+    'LinkedText',
+    'Permalink',
+    'Rating',
```

Comment:
Maybe we should alphabetize this while we're reformatting it anyway:

```suggestion
    'AddComment',
    'AddTag',
    'AjaxTab',
    'Cite',
    'DeleteComment',
    'DeleteTag',
    'Email',
    'Epub',
    'Explain',
    'Export',
    'Hold',
    'Home',
    'ILLRequest',
    'LinkedText',
    'PDF',
    'Permalink',
    'Rating',
    'RDF',
    'Save',
    'SMS',
    'StorageRetrievalRequest',
```